Utilities/COCommitDescriptor.h

```diff
@@ -309,3 +309,36 @@ extern NSString *kCOCommitMetadataShortDescription;
  * See also COCommitDescription Localization documentation section.
  */
 extern NSString *kCOCommitMetadataShortDescriptionArguments;
+/**
+ * The optional key that indicates the command group UUID used to synthesize
+ * the commit.
```

Comment:
Maybe "command group UUID which this commit selectively applies or undoes" would be clearer?